internal/distroregistry/distroregistry.go

```diff
@@ -21,6 +22,7 @@ var supportedDistros = []func() distro.Distro{
 	rhel8.New,
 	rhel84.New,
 	rhel84.NewCentos,
```

Comment:
Centos Should be moved to 8.5 now too, right?

Reply:
Yup, that's the plan.  I think we should do it in a separate PR though.  I wanted to keep this one short to see what people think of the way I wrote the image type definitions and pipelines.

Reply:
Makes sense. I guess we also need to implement all the other image types for 8.5 first too.